auth-web/src/components/pay/UnlinkedShortNameTable.vue

```diff
@@ -251,6 +247,10 @@ export default defineComponent({
       }
     ]
 
+    function selectAccount (account: EFTShortnameResponse) {
+      state.selectedAccount = account
```

Comment:
might be able to change
@account="selectAccount"

to
 `@account=state.selectedAccount = $event`